src/node/desktop/package.json

```diff
@@ -1,76 +1,76 @@
 {
-  "name": "rstudio",
-  "productName": "RStudio",
-  "version": "0.0.1",
-  "description": "RStudio",
-  "main": "./.webpack/main",
-  "repository": "https://github.com/rstudio/rstudio",
-  "author": "RStudio PBC",
-  "license": "AGPL-3.0-only",
-  "scripts": {
-    "clean": "ts-node scripts/clean.ts",
-    "lint": "eslint ./src ./test",
-    "make": "yarn run clean && electron-forge make",
-    "package": "electron-forge package",
-    "start": "electron-forge start",
-    "test": "electron-mocha -c --config ./test/unit/mocharc.json",
-    "testcover": "nyc electron-mocha -c --config ./test/unit/mocharc.json",
-    "testint": "mocha -c --config ./test/int/mocharc.json",
-    "testintwip": "mocha -c -g WIP  --config ./test/int/mocharc.json",
-    "testwip": "electron-mocha -c -g WIP --config ./test/unit/mocharc.json",
-    "win-package": "yarn run package && ts-node scripts/package-helper",
-    "format": "./scripts/create-prettier-ignore-file.sh && yarn prettier --write . --ignore-path .prettier_and_git_ignores"
-  },
-  "config": {
-    "forge": "./forge.config.js"
-  },
-  "devDependencies": {
-    "@electron-forge/cli": "^6.0.0-beta.61",
-    "@electron-forge/maker-deb": "^6.0.0-beta.61",
-    "@electron-forge/maker-dmg": "^6.0.0-beta.61",
-    "@electron-forge/maker-rpm": "^6.0.0-beta.61",
-    "@electron-forge/maker-squirrel": "^6.0.0-beta.61",
-    "@electron-forge/maker-zip": "^6.0.0-beta.61",
-    "@electron-forge/plugin-webpack": "^6.0.0-beta.61",
-    "@types/chai": "^4.2.22",
-    "@types/crc": "^3.4.0",
-    "@types/line-reader": "^0.0.34",
-    "@types/mocha": "^8.2.3",
-    "@types/node": "14.17.5",
-    "@types/sinon": "^10.0.6",
-    "@types/uuid": "^8.3.1",
-    "@typescript-eslint/eslint-plugin": "^5.4.0",
-    "@typescript-eslint/parser": "^5.4.0",
-    "@vercel/webpack-asset-relocator-loader": "1.7.0",
-    "chai": "^4.3.4",
-    "css-loader": "^6.5.1",
-    "electron": "^15.3.2",
-    "electron-mocha": "^10.1.0",
-    "eslint": "^8.2.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-config-standard": "^16.0.3",
-    "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-promise": "^5.1.1",
-    "fork-ts-checker-webpack-plugin": "^6.4.0",
-    "mocha": "^8.4.0",
-    "node-loader": "^2.0.0",
-    "nyc": "^15.1.0",
-    "playwright": "^1.16.3",
-    "prettier": "^2.4.1",
-    "recursive-copy": "^2.0.13",
-    "sinon": "^12.0.1",
-    "style-loader": "^3.3.1",
-    "ts-loader": "^9.2.6",
-    "ts-node": "^10.4.0",
-    "typescript": "^4.5.2"
-  },
-  "dependencies": {
-    "crc": "^3.8.0",
-    "electron-squirrel-startup": "^1.0.0",
-    "electron-store": "^8.0.1",
-    "line-reader": "^0.4.0",
-    "uuid": "^8.3.2"
-  }
+    "name": "rstudio",
```

Comment:
Did you change these to hard tabs on purpose (indentation)? We generally prefer spaces for indentation.